packages/frontend/webpack.config.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* global require, module */
+const {merge} = require('webpack-merge');
+const webpackCommonConfig = require('./Webpack5/webpack.common');
+const webpackDevelopmentConfig = require('./Webpack5/webpack.dev');
+const webpackProductionConfig = require('./Webpack5/webpack.prod');
+const { isProd } = require('./Webpack5/Utilities/Environment');
+
+/**
+ * @fileoverview Ponto de entrada para as configurações do webpack 5
+ * @see {@link https://www.npmjs.com/package/babel-register}
+ * @see {@link https://webpack.js.org/configuration/
+ * @module paquete/frontend/webpack.config
```

Comment:
Opa, acho que tá errado 
```suggestion
 * @module packets/frontend/webpack.config
```